ubuntudesign/wordpress-vanilla-theme
# Dyanmic header

#### What's this PR do?
This adds an update to ```header.php``` and adds a new dynamic navigation & search form. Also adds all title, description options + template description.

#### Where should the reviewer start?
Pull the branch set up a new WP instance locally and install the theme. Then create a couple standard pages.

#### How should this be manually tested?
Once setup view the site on your local development, the site should show the vanilla header + nav. The navigation should not allow sub categories to be shown in the navigation. Navigation should work fine in mobile.

#### Any background context you want to provide?
_n/a_

#### What are the relevant tickets?
_n/a_

#### Screenshots (if appropriate)
_expected outcome_
<img width="1402" alt="screen shot 2016-01-19 at 11 55 51" src="https://cloud.githubusercontent.com/assets/1201618/12417939/a54589ce-bea3-11e5-8ac4-a7d6fc09d72c.png">

#### Questions:
_n/a_
